charts/incubator/alfresco-dbp/values.yaml

```diff
@@ -204,7 +204,5 @@ alfresco-content-services:
     APP_CONFIG_OAUTH2_REDIRECT_LOGIN: "/workspace/"
     APP_CONFIG_OAUTH2_REDIRECT_LOGOUT: "/workspace/logout"
 
-alfresco-sync-service:
-  enabled: false
   alfresco-infrastructure:
```

Comment:
These two properties belong to syn service also. please remove them.

Reply:
https://github.com/Alfresco/alfresco-dbp-deployment/blob/845eac28278713a9710756caaa6f374f3be98858/charts/incubator/alfresco-dbp/values.yaml#L207-L208